src/assets/global.scss

```diff
@@ -40,8 +40,8 @@ $pill-blue-background: #3157b0;
 $pill-text-white: #f2f4fa;
 $pill-light-blue: #d5e0f2;
 $pill-text-blue: #324a88;
-$pill-green-background: #c2dfd1;
-$pill-green-text: #005328;
+$pill-green-background: #2f6e41;
```

Comment:
Question more than anything, but did this need to be changed/does it affect other places it's not supposed to?
I assume David has asked for a colour change, just checking